flink-connector-mysql-cdc/src/test/resources/ddl/column_type_test.sql

```diff
@@ -25,6 +25,8 @@ CREATE TABLE full_types (
     small_un_c SMALLINT UNSIGNED,
     int_c INTEGER ,
     int_un_c INTEGER UNSIGNED,
+    int11_c INT(11) ,
+    int11_un_c INT(11) UNSIGNED,
```

Comment:
One column to check `The type in db is INT, but defined BIGINT in Flink SQL` is enough,
Could you add `The type in db is FLOAT, but defined DOUBLE` in Flink SQL ?

Reply:
I think only `int11_c INT(11) ,` column is enough?

Reply:
Yes, I will remove it.

Reply:
> I think only `int11_c INT(11) ,` column is enough?

fixed it.